src/plugins.js

```diff
@@ -0,0 +1,19 @@
+const Ajv = require('ajv').default
+const semver = require('semver')
+
+const MIN_NODE_VERSION_KEYWORD = {
+  keyword: 'minNodeVersion',
+  validate: (minNodeVersion, { nodeVersion }) =>
+    semver.valid(minNodeVersion) && semver.valid(nodeVersion) && semver.gte(nodeVersion, minNodeVersion),
+}
+
+const getPlugins = function (plugins, { nodeVersion }) {
+  const ajv = new Ajv({})
+  ajv.addKeyword(MIN_NODE_VERSION_KEYWORD)
```

Comment:
Should those two lines be memoized?

`ajv` compiles JSON schemas under the hood, which is a little slow. It only does it the first time though (it is cached), so re-using the `ajv` instance between calls has a good performance advantage.

Reply:
Good point - I was probably too strict with not doing anything during `require` here